pkg/crc/constants/constants.go

```diff
@@ -174,24 +174,24 @@ func EnsureBaseDirectoriesExist() error {
 }
 
 func GetPublicKeyPath() string {
-	return filepath.Join(MachineInstanceDir, InstanceDirName(), "id_ecdsa.pub")
+	return filepath.Join(MachineInstancesDir, InstanceDirName(), "id_ecdsa.pub")
```

Comment:
I'd prefer to focus for now on having VM names, or machine dir name being `crc-$preset` instead of just `crc`, but keep for now "only one VM can exist at a given time".
This means most of this PR gets in, we can start making sure the change of names don't cause too many issues, ...
Then another PR can add support for multiple instance dir in the same subdir, multiple VMs, ...